tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+from bs4 import BeautifulSoup
+from pytest import fixture
+from selenium import webdriver
+
+from data import generate_form_data
+
+
+@fixture
```

Comment:
Обычно фикстуры не импортируют, а записывают так:
```suggestion
from selenium import webdriver

from data import generate_form_data


@pytest.fixture
```